modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureDM.testcase

```diff
@@ -38,118 +44,110 @@ definition {
-			dmDocumentTitle = "DM Document PDF");
-
-		var dmDocumentTitle = "DM Document PDF";
+			var dmDocumentTitle = "DM Document PDF";
+		}
 
-		Click(
-			dmDocumentTitle = ${dmDocumentTitle},
-			locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_SELECT_DOCUMENT");
+		task ("When the user goes to digital signature") {
+			DigitalSignature.documentDigitalSignature(documentlist = "DM Document PDF");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_COLLECT_BUTTON");
+		task ("Then confirm that you are on the page and delete the file") {
+			DigitalSignature.assertDigitalSignaturePage(assertPage = "New Digital Signature Envelope");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		DMDocument.deleteCP(dmDocumentTitle = "DM Document PDF");
+			DMDocument.deleteCP(dmDocumentTitle = "DM Document PDF");
+		}
 	}
 
-	@description = "This is a test for LPS-133993. Verify if its possible collect Digital Signature By vertical ellipsis"
+	@description = "LPS-133993. Verify if its possible collect Digital Signature By vertical ellipsis"
 	@priority = 5
 	test CanCollectDigitalSignatureByVerticalEllipsis {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+		task ("Given that user Test creates a document") {
+			DMDocument.addCP(
+				dmDocumentDescription = "DM Document Description",
+				dmDocumentFile = "Alice's Adventures in Wonderland.pdf",
+				dmDocumentTitle = "DM Document PDF");
 
-		DMDocument.addCP(
-			dmDocumentDescription = "DM Document Description",
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf",
-			dmDocumentTitle = "DM Document PDF");
-
-		var dmDocumentTitle = "DM Document PDF";
+			var dmDocumentTitle = "DM Document PDF";
+		}
 
-		Click.waitForMenuToggleJSClick(
-			key_rowEntry = ${dmDocumentTitle},
-			locator1 = "Icon#ROW_VERTICAL_ELLIPSIS");
+		task ("When the user goes to digital signature by ellipises") {
+			DigitalSignature.goToDigitalSignatureEllipsis(dmDocumentTitle = "DM Document PDF");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_ELLIPSIS_COLLECT_BUTTON");
+		task ("Then confirm that you are on the page and delete the file") {
+			DigitalSignature.assertDigitalSignaturePage(assertPage = "New Digital Signature Envelope");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		DMDocument.deleteCP(dmDocumentTitle = "DM Document PDF");
+			DMDocument.deleteCP(dmDocumentTitle = "DM Document PDF");
+		}
 	}
 
-	@description = "This is a test for LPS-135089. Verify if its possible get multiple signatures from diferent documents in DM"
+	@description = "LPS-135089. Verify if its possible get multiple signatures from diferent documents in DM"
 	@priority = 5
```

Comment:
Could you change the LPS on description to : "LPS-133105" because this is the story ticket for this automation, you can see story tickets [here](https://docs.google.com/spreadsheets/d/16UJDT4LxQX48zpslT8YuWGaMqWYMVr3W5JYJaDGqHVE/edit#gid=1778367802)